make-coe/make-coe.go

```diff
@@ -170,7 +191,7 @@ func (carina *MakeCOE) DeleteCluster(token string) (common.Cluster, error) {
 			}
 		}
 
-		return nil, errors.Wrap(err, fmt.Sprintf("[make-coe] Unable to delete cluster (%s)", token))
+		return nil, handleLibcarinaError(errors.Wrap(err, fmt.Sprintf("[make-coe] Unable to delete cluster (%s)", token)))
```

Comment:
Yup! Pretty much the same trick you had in `handlelibcarinaerror` so that we are sure that the check doesn't fail due to someone using `errors.WithStackTrace`, etc.